Themis/ViewModels/CorrectionSidebar/FeedbackViewModel.swift

```diff
@@ -37,10 +36,16 @@ public class FeedbackViewModel: ObservableObject {
         return feedback.score
     }
 
+    public func saveFeedback(feedback: Feedback) {
+        self.feedbacks.removeAll(where: { $0.id == feedback.id })
+        feedbacks.append(feedback)
```

Comment:
This results in the edited feedback being moved to the bottom of the list, which can be confusing.
Maybe try something like this (can't test on my own right now)
```suggestion
        if let index = feedbacks.firstIndex(where: { $0.id == feedback.id }) {
             feedbacks[index] = feedback
        }
```